stories/setup/Provider.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+import { createBrowserHistory } from 'history';
+import React from 'react';
+import { Provider as ReduxProvider } from 'react-redux';
+
+// Without core/polyfill we see the following error:
+// "regeneratorRuntime is not defined".
+import 'core/polyfill';
+
+import { dispatchClientMetadata } from '../../tests/unit/amo/helpers';
```

Comment:
I see an alias in the webpack config for `tests`. I think you can use that for this import path, right?

Reply:
sigh webpack... if I adjust the resolve info a bit yes I think it will work.

Reply:
I seem to be having issues with this too :/  It works but then lint and flow complain about not finding the module. If I don't put the resolve at all, then this then component complains about it 😕